### PR TITLE
Fixed several UI issues in safari, altered theme switching animation, fixed filetree arrow

### PR DESCRIPTION
**Pull request by JordanTheriault**

### CODE REVIEW

File: public/friendlycode/css/editor.less [hunk 1]
```diff
@@ -265,37 +264,32 @@
...
     margin-right: 10px;
     img {
-      position: relative;
-      left: -4px;
-    }
-    img#moon-green {
-      display: none;
+      position: absolute;
+      top: 100px;
+      left:165px;
```
Comment (humphd) [inline public/friendlycode/css/editor.less]: space after `:`

File: views/friendlycode/templates/nav-options.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="editor-nav">
 
   <!-- FILE TREE -->
-  <div class="filetree-pane-nav">
+  <div class="filetree-pane-nav" style="display:none;" class="hide">
```
Comment (humphd) [inline views/friendlycode/templates/nav-options.html]: Why do you need both the inline style attrib and the class?

**Comment by humphd** [comment]
A few small things, r+ on code.

File: views/friendlycode/templates/nav-options.html (hunk 2 above)
Comment (JordanTheriault) [inline views/friendlycode/templates/nav-options.html]: Not necessary, but added for consistency.
This element needs an inline display:none in order to hide it on load (the class adds the CSS too late, and creates a bad load visual).
 Everything else uses class="hide" except this element.

**Comment by JordanTheriault** [comment]
I'm going to wait until after the demo to merge.

**Pull request merged**